governance.md

```diff
@@ -1,14 +1,88 @@
-# Tekton Governance
+# Tekton Bootstrap Governance
 
-The Tekton project was donated to the [CDF](cd.foundation) in March 2019.
-Establishing governance is still a work in progress and for now, Tekton is
-overseen by the Knative Build WG.
+The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
 
-## Procedures
+Members are:
 
-### Creating a new repository
+* [Andrew Bayer](https://github.com/abayer) (CloudBees)
+* [Vincent Demeester](https://github.com/vdemeester) (RedHat)
+* [Christie Wilson](https://github.com/bobcatfish) (Google)
+* [Dan Lorenc](https://github.com/dlorenc) (Google)
+* [Kim Lewandowski](https://github.com/kimsterv) (Google)
 
-New repositories can be created in the `tektoncd` Github organization with approval from two
-maintainers (for now these can be found in [Github](https://github.com/orgs/tektoncd/teams/maintainers/members).
+The committee MUST:
+
+* Represent a cross-section of interests, not just one company
+* Balance technical, architectural, and governance expertise since its initial mission is the establishment of structure around contributions, community, and decision-making
+* Hold staggered terms, sufficient to ensure an orderly transition of power via elections as designed and implemented by the committee (see below for specific deliverables)
+* Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+* Communicate with the Continuous Delivery Foundation on a regular cadence
+
+## Governance Facilitator Role (optional)
```

Comment:
we don't have one of these atm tho right? so this is just describing it in case we use it later?

Reply:
Correct, we need to decide whether to use this role.